doc/BUILD-Debian.md

```diff
@@ -0,0 +1,74 @@
+1. install boost (apt-get)
```

Comment:
also add
```
sudo apt-get install g++ git-all libpng12-dev libjpeg-dev libeigen3-dev libboost-atomic-dev libboost-chrono-dev libboost-date-time-dev libboost-dev libboost-program-options-dev libboost-exception-dev libboost-filesystem-dev libboost-serialization-dev libboost-system-dev libboost-thread-dev libboost-timer-dev libtbb-dev
```
to make it quicker? This is the same we put in BUILD.md, and it fits here.